cmd/userns.go

```diff
@@ -91,6 +98,33 @@ func maybeReexecUsingUserNamespace(uidmap string, useNewuidmap bool, gidmap stri
 		}
 	}
 
+	// Log the ID maps we were started with.
+	UIDs, GIDs, err := unshare.GetHostIDMappings("")
+	if err != nil {
+		klog.Fatalf("Error reading current ID mappings: %v\n", err)
+	}
+	isNodeDefaultMapping := func(m []specs.LinuxIDMapping) bool {
+		return len(m) == 1 && m[0].ContainerID == 0 && m[0].HostID == 0 && m[0].Size == 0xffffffff
+	}
```

Comment:
Style nit - I prefer we move this to a regular function so that we can document with a unit test.

Reply:
Pulled it out as a separate function.